basestar-database-server/src/test/java/io/basestar/database/TestDatabaseServer.java

```diff
@@ -274,11 +279,15 @@ void updateSimple() throws Exception {
         final Map<String, Object> version2 = database.read(caller, SIMPLE, id, 2L).get();
         assertEquals(read, version2);
 
-        verify(emitter, times(1))
-                .emit(ObjectUpdatedEvent.of(SIMPLE, id, 1L, create, update));
+        log.info(" VERIFICATION PHASE OF UPDATE SIMPLE") ;
+
+        final InOrder inOrder = Mockito.inOrder(emitter);
+        inOrder.verify(emitter).emit(ObjectUpdatedEvent.of(SIMPLE, id, 1L, create, update));
+            //dont emmit events for identical values updates
+        // inOrder.verify(emitter).emit(any(ObjectUpdatedEvent.class));
+        inOrder.verifyNoMoreInteractions();
```

Comment:
@arcneon help needed. I'm concerned I may have either let slipped a subtle change of main code behaviour (bad!!) under the guise of a subtle change of Mockito semantics. 
Please double check.

In any case I probably ought to clean this up once we're satisfied either way.

Reply:
I'm not 100% sure what this comment/change refers to, when i run with jdk 11 i get a test pass with the original assert:

`
verify(emitter, times(1))
                .emit(ObjectUpdatedEvent.of(SIMPLE, id, 1L, create, update));
`
                
Were/are you getting an error with this assert? Can you add the error and some info about the build environment?